frida_tools/tracer.py

```diff
@@ -347,6 +355,26 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern (pattern) {
+    var m;
+    var f;
+    var res = pattern.split ('!');
+    
+    if (res.length == 1) {
+        m = '*';
+        f = res[0];
+    }
+    else {
+        m = (res[0] == '' ? '*' : res[0]);
+        f = (res[1] == '' ? '*' : res[1]);
+    }
+    
+    return {
+        module: m,
+        function: f
+    };
+}
```

Comment:
This looks like unrelated code?

Reply:
This is code added in a [previous pull request](https://github.com/frida/frida-tools/pull/26).  I don't know how it got into this PR, please ignore it.

Reply:
If you `git rebase origin/master`, it should clean up this PR.

Reply:
(And force-push after that: `git push -f`.)

Reply:
Thanks for the pointer.  I performed:

- git stash
- git rebase origin/master
- git stash apply

I hope this clears things up when I push.